vignettes/funbiogeo.Rmd

```diff
@@ -36,14 +39,7 @@ library("ggplot2")
 
 ## Provided data
 
-We are interested in mapping the functional traits of mammals of Europe.
-We used range maps data
-[from the IUCN](https://www.iucnredlist.org/resources/spatial-data-download)
-and trait data from the
-[PanTHERIA database](https://esapubs.org/archive/ecol/E090/184/)
-[@Jones_PanTHERIA_2009].
-The initial extraction scripts are available
-on [GitHub](https://github.com/FRBCesab/eumammals).
+We are interested in mapping the functional traits of 26 Rosacean tree species of Western European Mediterranean regions.
```

Comment:
We could copy the statement from man page to explain where the data are coming from.